main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       car
+//  action 1:   the car moves forward
+car.moveForward();
+//  action 2:   the car moves backward
+car.moveBackward();
+//  action 3:   the car plays radio
+car.playRadio();
+
+//  2) 
+//  Noun:       kid
+//  action 1:   the kid smiles
+kid.smile();
+//  action 2:   the kid cries
+kid.cry();
+//  action 3:   the kid sleeps
+kid.sleep();
 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       cat
+//  action 1:   the cat purr
+cat.purr();
+//  action 2:   the cat scratches
+cat.scratch();
+//  action 3:   the cat jumps
+cat.jump();
 
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       dog
+//  action 1:   the dog barks
+dog.bark();
+//  action 2:   the dog does tricks
+dog.doTricks();
+//  action 3:   the dog guards houses
+dog.guardHouses();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       cursor
+//  action 1:   the cursor blinks
+cursor.blink();
+//  action 2:   the cursor moves to the left
+cursor.moveToTheLeft();
+//  action 3:   the cursor moves to the right
+cursor.moveToTheRight();
 
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       bird
+//  action 1:   the bird flies 
+bird.fly();
+//  action 2:   the bird sings
+bird.sing();
+//  action 3:   the bird nests
+bird.nest();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

Reply:
Modified line 100 - 106 as requested